itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationOptimisedBootstrap.java

```diff
@@ -944,7 +975,8 @@ public void testOverwriteDuringBootstrap() throws Throwable {
 
     // Create some partitioned and non partitioned tables and do a dump & load.
     WarehouseInstance.Tuple tuple = primary.run("use " + primaryDbName)
-        .run("create table t1 (id int)")
+        .run("create table t1 (id int) clustered by(id) into 3 buckets stored as orc " +
+                "tblproperties (\"transactional\"=\"true\")")
```

Comment:
do we need to set this? Isn't is the default?

Reply:
In these tests the default table type is non-ACID?

Reply:
Default value of hive.strict.managed.tables is false. So, non-Acid managed tables are created if we don't specify the transactional property.